js/index.js

```diff
@@ -21,8 +21,17 @@ const chipSetsRadioGroup = document.getElementById("chipsets");
 const mainContainer = document.getElementById("mainContainer");
 let resizeTimeout = false;
 
-import { Transport } from './webserial.js'
-import { ESPLoader } from './ESPLoader.js'
+const usbPortFilters = [
+    { usbVendorId: 0x10c4, usbProductId: 0xea60 }, /* CP2102/CP2102N */
+    { usbVendorId: 0x0403, usbProductId: 0x6010 }, /* FT2232H */
+    { usbVendorId: 0x303a, usbProductId: 0x1001 }, /* Espressif USB_SERIAL_JTAG */
+    { usbVendorId: 0x303a, usbProductId: 0x1002 }, /* Espressif esp-usb-bridge firmware */
+    { usbVendorId: 0x303a, usbProductId: 0x0002 }, /* ESP32-S2 USB_CDC */
+    { usbVendorId: 0x303a, usbProductId: 0x0009 }, /* ESP32-S3 USB_CDC */
+];
+
+import { Transport } from '../node_modules/esptool-js/bundle.js'
+import { ESPLoader } from '../node_modules/esptool-js/bundle.js'
```

Comment:
`import { ESPLoader, Transport } from '../node_modules/esptool-js/bundle.js'` should work but you can also try:

```js
import * as esptooljs from "./bundle.js";
const ESPLoader = esptooljs.ESPLoader;
const Transport = esptooljs.Transport;
```